src/fides/api/models/privacy_experience.py

```diff
@@ -363,81 +580,19 @@ def get_privacy_notices_by_region_and_component(
     """
     return (
         db.query(PrivacyNotice)
-        .filter(PrivacyNotice.regions.overlap(regions))  # type: ignore[attr-defined]
+        .join(ExperienceNotices, PrivacyNotice.id == ExperienceNotices.notice_id)
+        .join(
+            PrivacyExperience,
+            PrivacyExperience.experience_config_id
+            == ExperienceNotices.experience_config_id,
+        )
         .filter(
-            or_(
-                and_(
-                    component == ComponentType.overlay,
-                    PrivacyNotice.displayed_in_overlay,
-                ),
-                and_(
-                    component == ComponentType.privacy_center,
-                    PrivacyNotice.displayed_in_privacy_center,
-                ),
-            )
+            PrivacyExperience.region.in_(regions),
+            PrivacyExperience.component == component,
         )
     )
 
 
-def upsert_privacy_experiences_after_notice_update(
-    db: Session, affected_regions: List[PrivacyNoticeRegion]
-) -> List[PrivacyExperience]:
-    """
-    Keeps Privacy Experiences in sync with *PrivacyNotices* changes.
-    Create or update "overlay" or "privacy center" PrivacyExperiences based on the PrivacyNotices in the "affected_regions".
-    To be called whenever PrivacyNotices are created or updated (pass in any regions that were potentially affected)
-
-    PrivacyExperiences should not be deleted.  It's okay if no notices are associated with an Experience.
-    """
```

Comment:
"PrivacyExperiences" aren't updated directly by the customer.  Previously, updates to Notices or updates to the PrivacyExperienceConfig could cause updates to the PrivacyExperiences in the background to make sure we always had a PrivacyExperience container for surfacing results to the end user.

Now, updates to PrivacyNotices do not affect which PrivacyExperiences should exist since "where" a notice is displayed is only defined on the PrivacyExperienceConfig.  Any places where notices are being created or updated no longer need to call this method.